app/controllers/bastion/bastion_controller.rb

```diff
@@ -2,6 +2,9 @@ module Bastion
   class BastionController < ::ApplicationController
     skip_before_filter :authorize
 
+    # prevent Foreman routes from being incorrectly generated (don't use the Bastion url_helpers)
+    include Rails.application.routes.url_helpers
```

Comment:
Interested to hear feedback here, but I think this will be alright.

When Foreman routes (ex: notification_recipients_path) were rendered through the Bastion layout they would double-dip the RELATIVE_URL_ROOT so we'd get broken paths: /foreman/foreman/notification_recipients. Including the top-level url_helpers prevents that from happening. If we wanted to reference bastion routes we could do so by setting the mount 'as' key to 'bastion' in mount_engine.rb so we can do `bastion.some_route_path` I have a feeling we would not ever need to do that, but that's how we would.